dvc/commands/plots.py

```diff
@@ -222,31 +222,21 @@ def run(self):
 
 
 class CmdPlotsTemplates(CmdBase):
-    TEMPLATES_CHOICES = [
-        "simple",
-        "linear",
-        "confusion",
-        "confusion_normalized",
-        "scatter",
-        "smooth",
-        "bar_horizontal_sorted",
-        "bar_horizontal",
-    ]
-
     def run(self):
-        from dvc_render.vega_templates import dump_templates
+        from dvc_render.vega_templates import TEMPLATES
 
         try:
-            out = (
-                os.path.join(os.getcwd(), self.args.out)
-                if self.args.out
-                else self.repo.plots.templates_dir
-            )
+            target = self.args.template
+            if target:
+                for template in TEMPLATES:
+                    if target == template.DEFAULT_NAME:
+                        ui.write(template.DEFAULT_CONTENT)
+                        return 0
+                raise ValueError(f"Unexpected template: {target}.")
```

Comment:
Just 💅 (`from dvc.exceptions import InvalidArgumentError`):
 
```suggestion
                raise InvalidArgumentError(f"Unexpected template: {target}.")
```